pkg/stanza/adapter/receiver.go

```diff
@@ -105,7 +104,9 @@ func (r *receiver) emitterLoop(ctx context.Context) {
 				continue
 			}
 
-			r.converter.Batch(e)
+			if err := r.converter.Batch(e); err != nil {
+				r.logger.Error("Could not add entry to batch")
```

Comment:
Should this log entry include original error?

Reply:
Good call. Added.